src/migrate/Migrator.js

```diff
@@ -42,6 +40,7 @@ const CONFIG_DEFAULT = Object.freeze({
   schemaName: null,
   directory: './migrations',
   disableTransactions: false,
+  sortDirsSeparately: false,
```

Comment:
The idea behind this flag is that while we are forced to sort all migrations at once for backwards compatibility (so that their execution order does not change), it is typically much more convenient to execute directories sequentially and only worry about migration ordering within the scope of a single directory (where it is very visible).